### PR TITLE
updated to release v2.0.0

### PR DESCRIPTION
**Pull request by deolank**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by jrgaray27 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (tbelmega) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]: The .github files are all being deleted in this PR? Aren't they a publishing requirement?

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,29 +4,34 @@
  The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
  and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [1.0.3] - 2024-08
-
-- Upgrade `axios` to mitigate [CVE-2024-39338](https://nvd.nist.gov/vuln/detail/CVE-2024-39338)
+## [2.0.0] - 2024-10-03
```
Comment (tbelmega) [inline CHANGELOG.md:7]: Are we deleting v1.0.3 from the Changelog? This looks like a mistake during resolving a merge conflict?

Comment (deolank) [reply CHANGELOG.md:7]: fixed

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (deolank) [reply .github/ISSUE_TEMPLATE/bug_report.md:1]: fixed

**Pull request merged**